dev/integration_tests/flavors/macos/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import FlutterMacOS
+import Cocoa
+import XCTest
+
+class RunnerTests: XCTestCase {
+
+  func testExample() {
+    // If you add code to the Runner application, consider adding tests here.
+    // See https://developer.apple.com/documentation/xctest for more information about using XCTest.
+  }
+
+}
```

Comment:
I wanted to delete this, but even when I delete the `RunnerTests` target via the Xcode UI (as well as the `RunnerTests` folder), there are still a bunch of references `RunnerTests` throughout the `dev/integration_tests/flavors/macos/Runner.xcodeproj/project.pbxproj` file. I'd rather just leave this alone, as I want the project to remain as close to a vanilla state (the one that a user creating a new project and following the documentation would have) as possible.